package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mands/nx-plugins",
-  "version": "0.3.7",
+  "version": "1.3.7",
```

```suggestion
  "version": "0.3.8",
```

Could you please update versions on both `package.json`